contrib/storage-cassandra/src/test/java/org/apache/drill/exec/store/cassandra/TestCassandraSuit.java

```diff
@@ -65,7 +66,12 @@ public static void tearDownCluster() {
 
   private static void startCassandra() {
     cassandra = new CassandraContainer<>("cassandra")
-        .withInitScript("queries.cql");
+                      .withInitScript("queries.cql")
+                      .withStartupTimeout(Duration.ofMinutes(3))
+                        .withEnv("CASSANDRA_SNITCH", "GossipingPropertyFileSnitch") // Tune Cassandra options for faster startup
+                          .withEnv("JVM_OPTS", "-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dcassandra.initial_token=0")
+                            .withEnv("HEAP_NEWSIZE", "128M")
+                              .withEnv("MAX_HEAP_SIZE", "1024M");
```

Comment:
We have not much memory on Github CI. Do we need this additional params for our test container? What about Splunk test container then?

Reply:
@vdiravka 
The Cassandra tests seem to keep timing out and breaking the CI.  My recollection of the Cassandra plugin was that Cassandra really does take a long time to start up, possibly more than 1 minute.  This was why we implemented it to preserve the connection as long as possible rather than creating tons of new connections. 

To my knowledge, the Splunk ones are working fine in the CI.

@luocooong  Thanks for this.  It makes me wonder, should we add the `GossipingPropertyFileSnitch` parameter to the Cassandra plugin?

Reply:
> The Cassandra tests seem to keep timing out and breaking the CI. My recollection of the Cassandra plugin was that Cassandra really does take a long time to start up, possibly more than 1 minute. This was why we implemented it to preserve the connection as long as possible rather than creating tons of new connections.

I am aware about timing out test of container for Cassandra plugin. I also fixed that several days ago in my branch. But I didn't merge it, because this is not only the one issue on CI and don't fix CI build. For instance the CI build is restarted several times for this PR and still not green.

--------------------------------------------------------------------------------------------------------------------------------------------
> To my knowledge, the Splunk ones are working fine in the CI.

My question was about memory options only in the above comment and `luocooong` answered that he is going to eliminate them.

--------------------------------------------------------------------------------------------------------------------------------------------
> It makes me wonder, should we add the GossipingPropertyFileSnitch parameter to the Cassandra plugin?

[GossipingPropertyFileSnitch](https://github.com/PaytmLabs/cassandra/blob/master/src/java/org/apache/cassandra/locator/GossipingPropertyFileSnitch.java) looks Cassandra specific code only.